providers/cloudflare/resources/cloudflare.lr

```diff
@@ -0,0 +1,234 @@
+option provider = "go.mondoo.com/cnquery/v11/providers/cloudflare"
+option go_package = "go.mondoo.com/cnquery/v11/providers/cloudflare/resources"
+
+// Cloudflare Provider
+cloudflare {
+	// List all zones
+	zones() []cloudflare.zone 
+
+	// List available accounts
+	accounts() []cloudflare.account
```

Comment:
you can put cloudflare into the expect.txt spellcheck file to quiet this

Reply:
thank you, updated